CMakeLists.txt

```diff
@@ -355,6 +355,13 @@ endif()
 ######################################################################################
 
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+option(HARDENING_ENABLE "Enable hardening against various attacks. May worsen performance" OFF)
+if(HARDENING_ENABLE)
```

Comment:
Can you change this to ENABLE_HARDENING (other flags seem to start with ENABLE) and add a sentence to CHANGES about this feature? Thanks.

Reply:
I don't think it makes any sense. `HARDENING_` prefix has been chosen because prefixes form collapsible groups in cmake-gui.

Reply:
Ok, I rarely use ccmake, didn't know. So the rest of the option names are wrong?

Reply:
What do you mean under `wrong`? The introduced options begin from `HARDENING_`. The old options haven't been touched.

Reply:
I just meant that if PoCL's old options are named ENABLE_XX they were added without knowing of such a semi-official (?) CMake config variable naming standard that is detected by the ccmake UI. So nevermind this comment, but the CHANGES is still valid.